NEWS.md

```diff
@@ -1,2 +1,6 @@
-## 1.0.0 Unreleased
-   * Project setup ([EDGDEMATIC-5](https://issues.folio.org/browse/EDGDEMATIC-5))
+## 1.0.0 - Released
```

Comment:
Links to JIRA stories/bugs in descending order (highest to lowest).

Reply:
Separate stories and bugs into separate lists

Reply:
Changed